### PR TITLE
encoding/raw: tests for inbound and outbound

### PR DESCRIPTION
**Pull request by abhinav**
CC @yarpc/golang

### CODE REVIEW

**Comment by breerly** [comment]
Less tests, more shipping :P

**Comment by breerly** [comment]
:shipit:

File: encoding/raw/inbound_test.go [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+				assert.Equal(t, []byte{1, 2, 3, 4, 5, 6}, body)
+				return []byte("hello"), nil, nil
+			},
+			wantBody: []byte("hello"),
+		},
+		{
+			procedure: "bar",
+			bodyChunks: [][]byte{
+				{1, 2, 3},
+				nil,
```
Comment (prashantv) [inline encoding/raw/inbound_test.go]: nit: maybe add a comment to the end saying this triggers error

Comment (abhinav) [inline encoding/raw/inbound_test.go]: 👍

**Comment by prashantv** [comment]
lgtm

**Pull request merged**